src/osd/ReplicatedBackend.h

```diff
@@ -431,7 +431,11 @@ class ReplicatedBackend : public PGBackend {
     ScrubMap &map,
     ScrubMapBuilder &pos,
     ScrubMap::object &o) override;
-  uint64_t be_get_ondisk_size(uint64_t logical_size) override { return logical_size; }
+
+  uint64_t be_get_ondisk_size(uint64_t logical_size) const override
```

Comment:
.h file, methods have opening brace on the same line as the signature.

Reply:
Probably should be final?

Reply:
Re the '.h' - I am using clang-format, and don't seem to have an easy way to
diff between 'cc' and 'h' files. And - I didn't see a clear reason for the distinction.
Anyway - will revert existing files manually.